ailab/ailab/views.py

```diff
@@ -0,0 +1,7 @@
+from future import absolute_import, division, print_function, unicode_literals
```

Comment:
The lint actually suggested the following... sorry about it

from __future__ import absolute_import, division, print_function, unicode_literals

Reply:
I see. github automatically removes the double underlines. It should be:
`from __future__ import absolute_import, division, print_function, unicode_literals`

Reply:
Sure. Will pay attention in the future.